models/comments-models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection")
+
+exports.selectCommentsByArticleId = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`, [article_id]).then(({rows})=>{
+        if(rows.length === 0){
+            return Promise.reject({status: 404, msg: '404: article does not exist'})
+        }
+        return rows;
```

Comment:
if this `rows.length === 0` does that mean that the article doesn't exist? What would your `rows` be if the article did exist but there were no comments for it yet? 🤔